docs/plugins/plugin-structure.md

```diff
@@ -149,3 +150,18 @@ Arguments: `message: Object` – object that you sent from `initializeAsync`
 Use this function to receive data back from your `initializeAsync` function. 
 
 Check `initializeAsync` and `onMessage` [example](./examples.md#using-initializeasync-and-onmessage)
+
+### `settings`
+Type: `Object`
+
+This object is used to specify settings that a plugin user can change. Each setting should include a `description` and a `type`. Other keys include:
+  * `description` -  `String`, description of the setting;
+  * `type` - `String`, used to decide element for rendering a setting.
+    * `string`
+    * `number`
+    * `bool`
+    * `option`
+    * `array`
+  * `defaultValue` - `Any`, default value for the setting;
```

Comment:
1. True
1. `multi` should actually only be applicable to `option`. `array` probably shouldn't ever be restricted to just one value.
1. The idea for `array` was to be the `creatable` version of `option`. I think they should all just be clearable by default, but that can be added.

I'll update the others